platform/qt/include/qmapbox.hpp

```diff
@@ -97,8 +97,6 @@ typedef void (*CustomLayerInitializeFunction)(void* context) ;
 typedef void (*CustomLayerRenderFunction)(void* context, const CustomLayerRenderParameters&);
 typedef void (*CustomLayerDeinitializeFunction)(void* context);
 
-Q_DECL_EXPORT void initializeGLExtensions();
```

Comment:
@jfirebaugh no need - we can safely remove these. We'll then update the Qt plugin accordingly 👍